lib/fluent/plugin/out_datadog.rb

```diff
@@ -12,21 +12,24 @@ class ConnectionFailure < StandardError; end
 
   # Register the plugin
   Fluent::Plugin.register_output('datadog', self)
+
   # Output settings
   config_param :use_json,           :bool,    :default => true
   config_param :include_tag_key,    :bool,    :default => false
-  config_param :tag_key,            :string,  :default => 'tag'
+  config_param :tag_key,            :string,  :default => 'fluentd_tag'
```

Comment:
Similarly, why changing the default tag key?

Reply:
Is it backward compatible ?